workitem/workitem_repository_blackbox_test.go

```diff
@@ -322,3 +322,11 @@ func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberNotFound() {
 	require.NotNil(s.T(), err)
 	assert.IsType(s.T(), errors.NotFoundError{}, errs.Cause(err))
 }
+
+func (s *workItemRepoBlackBoxTest) TestLoadBatchByID() {
+	fixtures := tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(5))
+	wis := []uuid.UUID{fixtures.WorkItems[1].ID, fixtures.WorkItems[1].ID, fixtures.WorkItems[2].ID, fixtures.WorkItems[2].ID}
+	res, err := s.repo.LoadBatchByID(s.Ctx, wis)
+	require.Nil(s.T(), err)
+	assert.Len(s.T(), res, 2)
```

Comment:
Great! This test creates 5 work items and batch loads 4 of them, while making sure that duplicates are loaded only once. Then we check that only 2 are loaded because 2 of the 4 where duplicates when loading. And we check that we don't load the whole 5 WIs. Great! But it is complicated and could need a comment ;)

Reply:
Thank you! Added comments in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/f267820f7c360f8da54ca40e6e34d6399f38f77e